src/js/containers/search/filters/naics/NAICSContainer.jsx

```diff
@@ -197,19 +197,24 @@ export class NAICSContainer extends React.Component {
     }
 
     onUncheck = (newChecked, uncheckedNode) => {
-        const [stagedNaicsFilters, newUnchecked] = decrementNaicsCountAndUpdateUnchecked(
-            uncheckedNode,
-            this.props.unchecked,
-            this.props.checked,
-            this.state.stagedNaicsFilters,
-            this.props.nodes
-        );
-
-        this.props.setUncheckedNaics(newUnchecked);
-        this.props.stageNaics(newChecked, newUnchecked, stagedNaicsFilters);
-        this.props.setCheckedNaics(newChecked);
-
-        this.setState({ stagedNaicsFilters });
+        if (uncheckedNode.checked) {
```

Comment:
I guess this is for if the unchecked is a parent?

Reply:
Kind of weird behavior; the checkboxTree invokes `onUncheck` when you check a half-checked node. But it does convey the checked state is still `checked: true`; whereas if it were a full check, then an uncheck, `checked: false` would be the state of this object.